ros_mscl/src/microstrain_3dm_node.cpp

```diff
@@ -6,15 +6,39 @@ This code is licensed under MIT license (see LICENSE file for details)
 
 */
 
-
 #include "microstrain_3dm.h"
-#include <ros/ros.h>
+#include "microstrain_diagnostic_updater.h"
+
+#include <rclcpp/rclcpp.hpp>
 
 int main(int argc, char **argv)
 {
-  ros::init(argc, argv, "ros_mscl_node");
-  Microstrain::Microstrain ustrain;
-  ustrain.run();
-  ros::shutdown();
+  rclcpp::init(argc, argv);
+
+  auto us = std::make_shared<Microstrain::Microstrain>();
```

Comment:
Little nit, can we rename this to something more descriptive?

Reply:
yep, will do!